cluster/test-deploy/gcp-crio/vars-origin.yaml

```diff
@@ -0,0 +1,108 @@
+# Package and target versions
+ 
+openshift_use_crio: true
```

Comment:
This is the file that is supposed to be the same across all variants.  You're only supposed to modify vars.yaml.

Reply:
okay, I will move this to vars.yml.

Reply:
What about the repo that we added to openshift_additional_repos?

Reply:
It might be getting merged. I'll try it out and read docs.